### ISSUE TITLE
How to avoid repeat destruction of xprt？（Bug）

### ISSUE THREAD

**Issue by Bingo0203** [nfs-ganesha/nfs-ganesha#1086]
Hello, gentleman

The problem now is that the umount operation disconnects the client from the server, and the “svc_vc_recv” function internally calls the “SVC_DESTROY” function, which destroys the xprt. At the same time, the ganesha process exits, and the “svc_xprt_shutdown” function duplicates the destruction of the xprt address, which results in the process going out of core. 

Is there a solution for this scenario?

**Comment by ffilz** [comment]
Hmm, in some sense this is a duplicate of #1073

**Comment by Bingo0203** [comment]
This indeed are the same issues. 
Label "Analyzing" means that  Currently there is no solution yet, it's still under analysis?

**Comment by ffilz** [comment]
Correct, Analyzing means we are still looking for root cause. Can we close this duplicate? If needed, add a comment to #1073
